cmd/launcher/launcher.go

```diff
@@ -368,6 +359,18 @@ func main() {
 		logger.Fatal(errors.Wrap(err, "launching osquery instance"))
 	}
 
+	ext.SetQuerier(runner)
```

Comment:
there's no info about this change outside the ADR - could you add comments about this and its implications?